docs/CHANGELOG.asciidoc

```diff
@@ -51,4 +51,8 @@ series decomposition. (See {pull}218[218].)
 
 Fix cause of "Bad density value..." log errors whilst forecasting. ({pull}207[207])
 
+Rules that trigger the `skip_model_update` action should also apply to the anomaly model.
+This fixes an issue where anomaly scores and probabilities on results that triggered
+the rule would reduce is they occurred frequently. (See {pull}222[#222].)
```

Comment:
According to the comment on line 6, `{pull}` should be `{ml-pull}`.

Reply:
I fixed this. However, the rest of the `pull`s in the changelog should also be `ml-pull`. I'll raise a PR to fix those in a separate commit.

Reply:
Thanks.  At the same time please could you also add some missing `#` symbols.  Looking at the old release notes the PR numbers that get rendered in HTML start with hashes, e.g. `#123`, but in our changelog not all the numbers have hashes.